File: Lv2/위장.py
def solution(clothes):
    answer = 1
    d = {}
    
    for c in clothes:
        if not(c[1] in d):
            d[c[1]] = 1
        else:
            d[c[1]] += 1
            
    for val in d.values():
        answer *= val + 1
    
    answer -= 1
    
    return answer

File: Lv2/위장.cpp
#include <string>
#include <vector>
#include <unordered_map>

using namespace std;

int solution(vector<vector<string>> clothes) {
    int answer = 1;
    unordered_map<string, int> m;
    
    for (auto c : clothes) {
        if (m.find(c.at(1)) == m.end()) 
            m.insert(make_pair(c.at(1), 1));
        else 
            m[c.at(1)]++;
    }
    
    for (auto e : m) 
        answer *= (e.second + 1);
    
    return answer - 1;
}